docs/csharp/language-reference/toc.yml

```
@@ -8,16 +8,17 @@
     items:
     - name: Overview
       href: keywords/index.md
-    - name: Types
+    - name: Builtin types
```

Though not sure about this suggestion:

```suggestion
    - name: Built-in types
```